spec/models/dialog_field_importer_spec.rb

```diff
@@ -4,14 +4,17 @@
   describe "#import_field" do
     let(:dialog_field) do
       {
-        "type"            => type,
-        "name"            => "Something",
-        "label"           => "Something else",
-        "resource_action" => resource_action,
-        "options"         => options
+        "type"                    => type,
+        "name"                    => "Something",
+        "label"                   => "Something else",
+        "resource_action"         => resource_action,
+        "options"                 => options,
+        "dialog_field_responders" => [responder]
       }
     end
 
+    let(:responder) { FactoryGirl.create(:dialog_field) }
```

Comment:
Do we need this? We're deleting the key anyway in the import process, I don't think there's any reason to actually create a dialog field in the test if we're just throwing it away.